pkgs/top-level/linux-kernels.nix

```diff
@@ -793,7 +793,7 @@ in
       linux_6_11_hardened = throw "linux 6.11 was removed because it has reached its end of life upstream";
       linux_6_13_hardened = throw "linux 6.13 was removed because it has reached its end of life upstream";
       linux_6_14_hardened = throw "linux 6.14 was removed because it has reached its end of life upstream";
-      linux_ham = throw "linux_ham has been removed in favour of the standard kernel packages";
+      linux_ham = recurseIntoAttrs (packagesFor kernels.linux_latest);
```

Comment:
Let's squash this into the original addition of the alias, so we don't have an intermediate state where it was a throw?